Wireframe/index.html

```diff
@@ -10,23 +10,44 @@
     <header>
       <h1>Wireframe</h1>
       <p>
-        This is the default, provided code and no changes have been made yet.
+        This page is a preview of wireframes - an introduction to semantic HTML.
       </p>
     </header>
     <main>
+      <!-- top article --> 
       <article>
         <img src="placeholder.svg" alt="" />
-        <h2>Title</h2>
+        <h2>Purpose of README</h2>
         <p>
-          Lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam,
-          voluptates. Quisquam, voluptates.
+           A README file is an introductory document for a project, providing essential information to users and developers. It's often the first thing they see, such as on GitHub, and its purpose is to make the project understandable, usable, and easily accessible. 
         </p>
-        <a href="">Read more</a>
+        <a href="https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-readmes">Read more</a>
       </article>
+
+      <article>
+        <img src="placeholder.svg" alt="" />
+        <h2>Wireframes</h2>
+        <p>
+          The primary purpose of a wireframe is to create a visual representation - a skeleton - of a website's layout.
+        </p>
+        <a href="https://balsamiq.com/learn/articles/what-are-wireframes/">Read more</a>
+      </article>
+
+      <article>
+        <img src="placeholder.svg" alt="" />
+        <h2>Git Branch</h2>
+        <p>
+          Git Branches allow us to work on different versions of our code simultaneously, like alternative timelines.It allow creating separate versions to try new things or multiple ideas in parallel without risking the original codebase.
+
+If we make changes on one branch, they do not impact the other branches until we merge or integrate the changes.
+        </p>
+        <a href="https://codeinstitute.net/blog/git-branches/">Read more</a>
+      </article>
+
     </main>
     <footer>
       <p>
-        This is the default, provided code and no changes have been made yet.
+        This is the footer.
```

Comment:
When scrolling towards your page, the footer also moves with the screen, causing the footer text to overlap with other components on your webpage, which makes it a bit unappealing to viewers. Could you please have a look at how this can be fixed or improved?

<img width="486" alt="Screenshot 2025-05-21 at 08 08 28" src="https://github.com/user-attachments/assets/7f6a32a7-6c86-4ecd-94bf-8ec696c81da6" />